docs(blog): personal vs univeral voice ai models post (#162)
spokestack/spokestack-website: content/blog/what-are-personal-ai-models/index.md

---
title: What Are Personal AI Models?
date: '2021-03-25'
description: Spokestack Maker introduces sophisticated AutoML tools for creating personal AI models. What are they, and how should they be used?
tags: News, Product
author: noel
draft: true
hero:
---

> The [Spokestack Maker](/account/settings#billing) service is designed for two audiences: hobbyists who want to personalize their projects, and developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality TTS voice. This is a great way to test drive the technology without breaking the bank.

Reviewer: This will work for now, although eventually it might make more sense to link to a feature page...

> The [Spokestack Maker](/account/settings#billing) service is designed for two audiences: hobbyists who want to personalize their projects, and developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality TTS voice. This is a great way to test drive the technology without breaking the bank.

Reviewer: The current link for "Spokestack Maker" will work for now, although maybe eventually we link to the feature page?

Author: Yeah, i think that's the case.

> The [Spokestack Maker](/account/settings#billing) service is designed for two audiences: hobbyists who want to personalize their projects, and developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality TTS voice. This is a great way to test drive the technology without breaking the bank.

Reviewer: ```suggestion
The [Spokestack Maker](/account/settings#billing) service is designed for two audiences: (1) hobbyists who want to personalize their projects, and (2) developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality text-to-speech (TTS) voice. This is a great way to test drive the technology without breaking the bank.
```

> ## What are the differences between "personal" and "universal" voice AI models?

Reviewer: With [Wikipedia style](https://titlecaseconverter.com/) casing:

```suggestion
## What are the Differences Between "Personal" and "Universal" Voice AI Models?
```

> Like any neural models, the personal models created with these tools are only as good as the data they're trained on. In other words, a wake word created in Maker, will not respond to everyone else in the world as well as it does to you if it's shipped in a mobile app — it will only work reliably for voices that resemble the one(s) it was trained with.

Reviewer 2: ```suggestion
Like any neural models, the personal models created with these tools are only as good as the data they're trained on. In other words, a wake word created in Maker will not respond to everyone else in the world as well as it does to you if it's shipped in a mobile app — it will only work reliably for voices that resemble the one(s) it was trained with.
```

For a home automation project, though, this specificity might be a feature — have you ever wished that your smart speaker didn't listen to the kids quite so well? (Of course, if you _do_ want your DIY smart speaker to listen to everyone, you can call the kids over to the computer and have them help you train it!)

> Similarly with Maker personal TTS voice cloning; in just 75 sentences you can train a clone of your voice, but its pronunciation and similarity to the human it's mimicking isn't going to be quite as good as a voice trained on dozens of hours of audio recorded in a studio. It'll still be plenty recognizable — check out the voices on our showcase along with their training criteria — but know that you could do better with more data.

Reviewer 2: ```suggestion
Similarly with Maker personal TTS voice cloning; in just 75 sentences you can train a clone of your voice, but its pronunciation and similarity to the human it's mimicking aren't going to be quite as good as a voice trained on dozens of hours of audio recorded in a studio. It'll still be plenty recognizable — check out the voices on our showcase along with their training criteria — but know that you could do better with more data. If you happen to have good recording equipment and a quiet environment, we do have thousands of scripts queued up in the training tool, so go ahead and record as much as you have patience for. Our automatic model trainer will adjust to make the best use of the data you provide it.
```

> Similarly with Maker personal TTS voice cloning; in just 75 sentences you can train a clone of your voice, but its pronunciation and similarity to the human it's mimicking isn't going to be quite as good as a voice trained on dozens of hours of audio recorded in a studio. It'll still be plenty recognizable — check out the voices on our showcase along with their training criteria — but know that you could do better with more data.

Reviewer: ```suggestion
Similarly, with Maker personal TTS you can train a clone of your voice in just 75 sentences. However, its pronunciation and similarity to the human it's mimicking isn't going to be quite as good as a voice trained on dozens of hours of audio recorded in a studio. It'll still be plenty recognizable — check out the voices on our showcase along with their training criteria — but high-quality samples, and lots of them, always produces better models than low-resource data.
```

> With Spokestack Maker, you'll still be able to use the NLU model trainer and TTS showcase, but you'll also get access to three new tools for training your own [wake word](/docs/concepts/wakeword), [keyword](/docs/concepts/keyword), and [TTS](/docs/concepts/tts) models using data you record yourself.

Reviewer 2: In the context of this paragraph, it seems like these should be links to the tools themselves, but I know that doesn't work for free-tier subscribers. Might be something to think about in the next redesign—sending non-subscribers to tool-specific splash pages/previews or something.
